Update getting-started.md

corinthian13 opened the pull request:
Lines 12 - 15.

More clarity on how to add PATH entry for Alire ecosystem client installed on a user's local Linux machine.  
PATH value was missing the double quotes that _may_ be needed when file paths contain spaces, e.g. /home/me/ada stuff/alr/bin.

corinthian13 committed:
Update getting-started.md

Lines 12 - 15.

More clarity on how to add PATH entry for Alire ecosystem client installed on a user's local Linux machine. 
PATH value was missing the required double quotes.

Fabien-Chouteau commented on doc/getting-started.md:
```diff
@@ -9,9 +9,9 @@ repository](https://github.com/alire-project/alire/releases).
 On Linux, `Alire` is simply provided in an archive.
 
-Once the archive is extracted you have to add `alr` in the environment `PATH`:
+Once the archive is extracted you have to add `alr` in the environment `PATH`in the .bashrc file of the user's home directory:
```
I am not sure everyone will want to put that in `.bashrc`. I personally prefer to set the PATH every time I want to use Alire.

corinthian13 replied:
Then I suppose it's a case of an _either ... or ..._ depending on the user's choice.
Personally I would see no danger in having this additional path entry and a lot of convenience in retaining it.

mosteo replied:
This could be reworded as to indicate that it's an optional possibility and not a requirement for use.

corinthian13 replied:
Edited as per mosteo suggestion.

jquorning replied:
I guess this belongs in a **setup-environment.md** document.

corinthian13 replied:
@jquorning 

Why not where it is right now, i.e. the "Getting Started" page ?
https://github.com/alire-project/alire/blob/master/doc/getting-started.md

jquorning replied:
> Why not where it is right now, i.e. the "Getting Started" page ? 

I guess the environment is a big matrix of combinations of terminals, character sets, shells, configurations, etcetera not controlled by Alire. Especially on Linux. IMO Alire  should not try to adapt to all kinds of specialized use cases internally.

corinthian13 committed:
Update getting-started.md

To offer users a choice of adding the Alire client environment to their PATH environment either just for their current terminal session or for future sessions also.

corinthian13 committed:
Update getting-started.md

Put PATH entries into .profile rather than .bashrc since some users may not be using the bash shell, i.e. may be using csh, zsh, fsh, etc.

mosteo commented on index/ad/adacl_eastrings/adacl_eastrings-6.2.0.toml:
```diff
@@ -0,0 +1,53 @@
+validation.runtime_checks   = "Everything"
+
+[[depends-on]]
+gnat_native                 = "^14.2"
```
Same comment as in #1342

krischik replied:
The crate will only work for Windows, Linux and macOS and I'm indeed unsure about about cross compiler. And then there was a problem with macOS picking a cross compiler even when native compilation was requested. See:

https://matrix.to/#/!DNeiNadAgLpbmdDKov:gitter.im/$MN8KphfRswrJ8Bw66FHWL2mVM_Xyd1eOgtZ9jFVrbSI?via=gitter.im (of you can make the link work)

So, yes, that was intentional.
